x-pack/plugins/cases/server/plugin.ts

```diff
@@ -163,6 +166,12 @@ export class CasePlugin {
       scheduleCasesTelemetryTask(plugins.taskManager, this.logger);
     }
 
+    this.userProfileService.initialize({
```

Comment:
Do we need to initialize it before the registration?

Reply:
My mistake, I though this line of code was in the `setup` method.

Reply:
We talked offline, the `initialize` is done within the `start` and the reference is needed in the `setup` so we'll keep it as it is for now.